src/platform/datapath_winuser.c

```diff
@@ -770,19 +770,14 @@ QuicDataPathInitialize(
 
 _IRQL_requires_max_(PASSIVE_LEVEL)
 void
-QuicDataPathUninitialize(
+QuicDataPathShutdownWorkers(
     _In_ QUIC_DATAPATH* Datapath
     )
 {
     if (Datapath == NULL) {
         return;
     }
```

Comment:
Do we ever actually call with a NULL input? According to the SAL annotation, it's not optional. So I think we can just remove (and not add) all these checks.